shared/src/commonMain/kotlin/com/br/kmmdemo/theme/Typography.kt

```diff
@@ -0,0 +1,59 @@
+package com.br.kmmdemo.theme
+
+import androidx.compose.material3.Typography
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+
+// FONT FAMILY CAN ONLY BE SET AT THE COMPOSABLE WHEN USING MOKO RESOURCES AT THIS TIME
+
+val kmpTypography = Typography(
```

Comment:
Change this to a factory function that is a composable. Looking at theme code that uses this val it is used from within a composable.  So it should work out.